pkgs/by-name/je/jerryscript/package.nix

```
@@ -0,0 +1,93 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  python3,
+  validatePkgConfig,
+  versionCheckHook,
+  testers,
+  nix-update-script,
+
+  cmdline ? !stdenv.hostPlatform.isNone,
+  math ? false,
```

```suggestion
  jerryCmdline ? !stdenv.hostPlatform.isNone,
  jerryMath ? false,
```

Prevent overlapping package name